recipes/leptonica/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "leptonica" %}
+{% set version = "1.76.0" %}
+{% set sha256 = "ff98ffef6103cda0e400676f81113a8157569ac0f453c6e03a1a27b305ba25a7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DanBloomberg/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [osx or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cmake
```

Comment:
This is using `pkg-config` as well. Currently it is reaching for the system copy, but we can provide our own (on macOS too). So please add it here.

Based on these log lines.

```
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.23") 
```

ref: https://circleci.com/gh/conda-forge/staged-recipes/27445